boards/shields/adafruit_neopixel_grid_bff/doc/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _adafruit_neopixel_grid_bff:
+
+Adafruit 5x5 NeoPixel Grid BFF
+##############################
+
+Overview
+********
+
+The `Adafruit 5x5 NeoPixel Grid BFF`_ is a 5x5 grid of WS2812B RGB LEDs,
+which Adafruit refers to as "neopixels". The BFF series of shields are
+designed to be compatible with any Adafruit Qt Py or Seed Studio Xiao
+board.
+
+.. figure:: adafruit_neopixel_grid_bff.jpg
+   :align: center
+   :alt: Adafruit 5x5 NeoPixel Grid BFF
+
+   Adafruit 5x5 NeoPixel Grid BFF (Credit: Adafruit)
+
+Pin Assignments
+===============
+
++-----------------------+---------------------------------------------+
+| Shield Connector Pin  | Function                                    |
++=======================+=============================================+
+| A3                    | WS2812B driver pin [1]_                     |
++-----------------------+---------------------------------------------+
+
+.. [1] This is the default pin when the board is delivered as is by Adafruit.
+       It can be changed away from the default using the onboard solder jumpers,
+       but this shield will no longer work without a matching device tree update.
+
+Programming
+***********
+
+LED Strip Example
+=================
+
+This example lights up each LED along the strip one at a time; first red, then
+blue, then green. After getting the end of the strip, the pattern repeats.
+
+Set ``-DSHIELD=adafruit_neopixel_grid_bff`` when you invoke ``west build``. For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/led_ws2812
+   :board: adafruit_qt_py_rp2040
+   :shield: adafruit_neopixel_grid_bff
+   :goals: build
+
+LED Display Matrix Example
+==========================
+
+This example puts a test image on the LED display, demonstrating how to use the
+LED Matrix display driver. This driver allows you to display small images on
+the 5x5 LED display.
```

Comment:
```suggestion
```

Same as before.
It would be better IMO to use this intro paragraph to explain that the shield provides an additional optional overlay for adding a display compatible, that behaves just as any other display in Zephyr, since otherwise it's not really obvious to someone new to this, while in fact this is a very cool feature we don't want people to miss :)

Another option would be that this description could be directly add to the Overview section of the doc page, i.e. something like ".... the overlays provided by this shield enable the LED strip corresponding to the matrix and, optionally, can also expose it as a standard display controller...." 

The warning that follows is definitely a good addition and must be kept.

Reply:
Paragraph removed.